### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[bug] Breif Description"
```
Comment (Thalhammer) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: Brief

File: .github/ISSUE_TEMPLATE/support-question.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Support Question
+about: Open an issue describing the topic
+title: "[question] Breif Description"
```
Comment (Thalhammer) [inline .github/ISSUE_TEMPLATE/support-question.md:4]: Brief

**Review by Thalhammer — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (prince-chrismc) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]:
```suggestion
title: "[bug] Brief Description"
```

**Comment by prince-chrismc** [comment]
I want to take a look for some other examples, this way it should be easier to resolve after 🤞

**Review by Thalhammer — APPROVED** [review]

**Pull request merged**